models/db.js

```diff
@@ -39,6 +38,9 @@ var UserSchema = new Schema({
     us_contact: {type: String, required: true},
     us_user_creation_date: {type: Date, default: Date.now()},
     us_state: {type: String, default: 'pending'},
+    us_address: {type: String, default: 'pending'},
```

Comment:
remove default attribute and replace with `required: true` for all mandatory fields.  The absence of the field will have a default of pending instead of throwing an error for missing field in request.